apps/test/unit/templates/teacherDashboard/teacherSectionsReduxTest.js

```diff
@@ -744,22 +744,22 @@ describe('teacherSectionsRedux', () => {
     it('updates an edited section in the section map on success', () => {
       const sectionId = 12;
       store.dispatch(beginEditingSection(sectionId));
-      store.dispatch(editSectionProperties({grade: 'K'}));
+      store.dispatch(editSectionProperties({grades: ['K']}));
 
       // Set up matching server response
       server.respondWith(
         'PATCH',
         `/dashboardapi/sections/${sectionId}`,
-        successResponse({grade: 'K'})
+        successResponse({grades: ['K']})
       );
 
       store.dispatch(finishEditingSection());
-      expect(state().sectionBeingEdited).to.have.property('grade', 'K');
-      expect(state().sections[sectionId]).to.have.property('grade', '11');
+      expect(state().sectionBeingEdited.grades).to.eql(['K']);
+      expect(state().sections[sectionId].grades).to.eql(['11']);
```

Comment:
there's supposedly support for `.deep.property`, but it doesn't seem to work with array comparison